src/Sylius/Bundle/PromotionBundle/DependencyInjection/Configuration.php

```diff
@@ -148,6 +150,23 @@ private function addResourcesSection(ArrayNodeDefinition $node): void
                                 ->end()
                             ->end()
                         ->end()
+                        ->arrayNode('catalog_promotion_action')
+                            ->addDefaultsIfNotSet()
+                            ->children()
+                                ->variableNode('options')->end()
+                                ->arrayNode('classes')
+                                    ->addDefaultsIfNotSet()
+                                    ->children()
+                                        ->scalarNode('model')->defaultValue(CatalogPromotionAction::class)->cannotBeEmpty()->end()
+                                        ->scalarNode('interface')->defaultValue(CatalogPromotionActionInterface::class)->cannotBeEmpty()->end()
```

Comment:
Isn't needed anymore

```suggestion
```

Reply:
Indeed 👍 but to be consistent with other configurations I would leave it for now and remove altogether later 🖖